scripts/syschdemd.nix

```diff
@@ -43,7 +43,7 @@ mkWrappedScript {
     "/run/wrappers" # mount
     coreutils
     daemonize
-    glibc # getent
+    getent
```

Comment:
we could also use glibc.getent but using getent is more cross friendly even if that does not matter here https://github.com/NixOS/nixpkgs/blob/3a671cd64d03f4e55eb6c817d6294910e0ccc133/pkgs/top-level/unixtools.nix#L74-L78